src/Adapter/TaxRulesGroup/CommandHandler/AddTaxRulesGroupHandler.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\TaxRulesGroup\CommandHandler;
+
+use PrestaShop\PrestaShop\Adapter\TaxRulesGroup\AbstractTaxRulesGroupHandler;
+use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Command\AddTaxRulesGroupCommand;
+use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\CommandHandler\AddTaxRulesGroupHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Exception\CannotAddTaxRulesGroupException;
+use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Exception\TaxRulesGroupException;
+use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\ValueObject\TaxRulesGroupId;
+use PrestaShopException;
+use TaxRulesGroup;
+
+/**
+ * Handles tax rules group addition
+ */
+class AddTaxRulesGroupHandler extends AbstractTaxRulesGroupHandler implements AddTaxRulesGroupHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(AddTaxRulesGroupCommand $command): TaxRulesGroupId
+    {
+        $taxRulesGroup = new TaxRulesGroup();
+        $taxRulesGroup->name = $command->getName();
+        $taxRulesGroup->active = $command->isEnabled();
+
+        try {
+            if (false === $taxRulesGroup->validateFields(false)
+                || false === $taxRulesGroup->validateFieldsLang(false)
+            ) {
```

Comment:
as described in psr-12 standards (https://www.php-fig.org/psr/psr-12/#51-if-elseif-else) it should probably look like this:
```suggestion
            if (
                false === $taxRulesGroup->validateFields(false)
                || false === $taxRulesGroup->validateFieldsLang(false)
            ) {
```

Reply:
Fixed